vendor/stmicroelectronics/nucleo-wl55jc-profile-cn470.yaml

```diff
@@ -0,0 +1,12 @@
+supportsClassB: true
+classBTimeout: 250
+pingSlotPeriod: 512
+pingSlotDataRateIndex: 2
+pingSlotFrequency: 0
```

Comment:
This doesn't seem right

Reply:
Are you talking about the ping_slot field, related to the previous comment, or is it another element specific to this file?

Reply:
I'm referring to the frequency `0`.

Reply:
The beacon/ping slot frequency is not fixed for CN470, AU915 and US915. Here, the Regional Parameter definition for the Class B configuration of CN470:
`Example: the first beacon will be transmitted on 508.3Mhz, the second on 508.5MHz, the 9th beacon will be on 508.3Mhz again.`
Each 128seconds, the beacon and ping slot configuration use a new channel, defined by 2 equations:
Beacon : In Regional Parameters specification, chapters xxx Class B beacon 
`BeaconChannel = ...`
Ping Slot : In L2 specification (v1.0.3) chapter 16.2 Frequency-hopping beacon transmission
`Class B downlink channel = ...`

So, its not possible to set only one value in this configuration. If you have an interesting idea in order to provide the whole table of 8 frequencies corresponding to the 8 channels of Class B, I'm interested :)

Reply:
Hmm, right, in these bands it doesn't make sense to specify this.